pkg/linter/linter.go

```diff
@@ -68,7 +73,7 @@ func (l Linter) WithInputModules(input *rules.Input) Linter {
 	return l
 }
 
-// WithAddedBundle adds a bundle of rules and data to include in evaluation.
+// WithAddedBundle overrides the default Regal rules with the provided bundle.
```

Comment:
It will add to the rules, not override them.

Reply:
Oops, have fixed that now. I've also opted to use this in our tests: d98d51095527fdc76474169b48587b362100d5a1